// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/es6/Symbols/symbolProperty58.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -3,22 +3,7 @@
 //// [symbolProperty58.d.ts]
 interface SymbolConstructor {
     foo: string;
 }
-declare var obj: invalid;
-
-/// [Errors] ////
-
-symbolProperty58.ts(6,5): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-
-
-==== symbolProperty58.ts (1 errors) ====
-    interface SymbolConstructor {
-        foo: string;
-    }
-    
-    var obj = {
-        [Symbol.foo]: 0
-        ~~~~~~~~~~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-!!! related TS9027 symbolProperty58.ts:5:5: Add a type annotation to the variable obj.
-    }
\ No newline at end of file
+declare var obj: {
+    [Symbol.foo]: number;
+};
